os/drivers/lcd/mipi_lcd.c

```diff
@@ -38,6 +38,11 @@
 #define CONFIG_LCD_MAXPOWER 100
 #endif
 
+#ifdef CONFIG_LCD_ST7701SN
+uint8_t *lcd_buffer_1 = NULL, *lcd_buffer_2 = NULL;
+bool is_lcd_buffer_1 = false;
+#endif
```

Comment:
1. Let's make a new config for sw orientation or sw rotation.
2. Let's add `static`.
3. Let's define each at each line.
```
uint8_t *lcd...._1 = NULL;
uint8_t *lcd..._2 = NULL;
bool .... = false;
```
4. How about below?
```
#define NUM_OF_LCD_BUFFER  (2)
uint8_t lcd_buffer[NUM_OF_LCD_BUFFER] = { NULL, NULL };
uint8_t lcd_buffer_index = 0;
```

Reply:
done.

Reply:
1st and 2nd are not done.
For 1st, you use the config for ST7701SN. But it's not a feature of that. Let's make NEW config for SW rotation and use it.